src/clinical/donor-repo.ts

```diff
@@ -190,7 +183,11 @@ export const donorDao: DonorRepository = {
   ): Promise<DeepReadonly<{ donors: Donor[]; totalDonors: number }>> {
     const { sort, pageSize, entityTypes, donorIds, submitterDonorIds, completionState } = query;
 
-    const projection = [...DONOR_ENTITY_CORE_FIELDS, ...entityTypes].join(' ');
+    const projection = [
+      ...DONOR_ENTITY_CORE_FIELDS,
+      ...entityTypes,
+      ...requiredEntities(entityTypes),
```

Comment:
ok so I see this function is to get the required document fields for the projection, I'd call it `getRequiredDonorDocumentFieldsForEntityTypes()` a bit more verbose but clearer

Reply:
Sure I'll use `getRequiredDonorFieldsForEntityTypes` just to keep it a little concise, I think 'document' is understood